pkg/cloudprovider/openstack/openstack.go

```diff
@@ -480,6 +480,7 @@ func (lb *LoadBalancer) GetTCPLoadBalancer(name, region string) (endpoint string
 // a list of regions (from config) and query/create loadbalancers in
 // each region.
 
+// TODO: Verify this is idempotent
```

Comment:
There's relevant discussion ongoing around this:
https://github.com/GoogleCloudPlatform/kubernetes/pull/7852#discussion_r29817700
https://github.com/GoogleCloudPlatform/kubernetes/issues/8352